images/ami/ubuntu-focal.yaml

```diff
@@ -0,0 +1,14 @@
+download_images: true
+packer:
+  ami_filter_name: "ubuntu/images/hvm-ssd/ubuntu-focal-20.04-amd64-server*"
+  ami_filter_owners: "099720109477"
+  source_ami: ""
+  distribution: "Ubuntu"
+  distribution_version: "20.04"
+  ssh_username: "ubuntu"
+  root_device_name: "/dev/sda1"
+  volume_size: "15"
+
+build_name: "ubuntu-focal"
```

Comment:
can we make it "ubuntu-20" to be in line with the other distros?

Reply:
ubuntu 20.04 or 20.14?